snakesist/exist_client.py

```diff
@@ -165,35 +178,50 @@ class ExistClient:
     :param user: username
     :param password: password
     :param prefix: configured path prefix for the eXist instance
+    :param root_collection: a path to a collection which will be used as root for all document paths
     :param parser: an lxml etree.XMLParser instance to parse query results
     """
 
     def __init__(
-            self,
-            host: str = DEFAULT_HOST,
-            port: int = DEFAULT_PORT,
-            user: str = DEFAULT_USER,
-            password: str = DEFAULT_PASSWORD,
-            prefix: str = "exist",
-            parser: etree.XMLParser = DEFAULT_PARSER
+        self,
+        host: str = DEFAULT_HOST,
+        port: int = DEFAULT_PORT,
+        user: str = DEFAULT_USER,
+        password: str = DEFAULT_PASSWORD,
+        prefix: str = "exist",
```

Comment:
regarding `prefix` and `root_collection`, see this: https://github.com/03b8/snakesist/pull/23#issuecomment-651773650
but i guess that will solve itself in the process.

Reply:
ok. i'll just leave this as is until that conversation is resolved

Reply:
i meant: leave it as is in this pr. if changes are required, they'll come up by themselves.